packages/mui-system/src/createTheme/createSpacing.ts

```diff
@@ -13,7 +13,7 @@ export type SpacingArgument = number | string;
 // We express the difference with variable names.
 export interface Spacing {
   (): string;
-  (value: number): string;
+  (value: number | string): string;
```

Comment:
```suggestion
  (value: SpacingArgument): string;
```

Reply:
TBH, I would prefer `string | number` than `SpacingArgument` but since it's already there, let's make it consistent for now.